packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -419,13 +431,17 @@ const holdPageTrampoline = (): string => {
   return 'call';
 };
 
-const unsupportedEnvironmentPageTrampoline = (): string => {
-  /* @conditional-compile-remove(unsupported-browser) */
-  return 'unsupportedEnvironment';
-  return 'call';
-};
-
-const getQueryOptions = (options: { /* @conditional-compile-remove(rooms) */ role?: Role }): PermissionConstraints => {
+const getQueryOptions = (options: {
+  /* @conditional-compile-remove(rooms) */ role?: Role;
+  /* @conditional-compile-remove(call-readiness) */ callReadinessOptedIn?: boolean;
+}): PermissionConstraints => {
+  /* @conditional-compile-remove(call-readiness) */
+  if (options.callReadinessOptedIn) {
```

Comment:
this looks unintentional, was this meant to be added in or artefact after merge from main?

Reply:
this was a artifact from main in a conflict resolution